ci-operator/config/tnozicka/openshift-acme/master.yaml

```diff
@@ -14,10 +24,28 @@ images:
       as:
       - bin
   to: openshift-acme
+
 promotion:
   cluster: https://api.ci.openshift.org
   namespace: ci
   tag: latest
+
+tag_specification:
```

Comment:
Is adding this enough or it requires modifications of https://github.com/openshift/release/blob/2f38d36898dda73d24dae62d5e9b25dc186f5d0f/ci-operator/jobs/tnozicka/openshift-acme/manual-presubmits.yaml ?